Unable to start lsc with systemd units

Hello,

I tried to start the service "lsc-async.service" but it fails. Here are the error logs:

```
Jul 28 13:55:33 systemd[1]: Starting LSC server...
-- Subject: Unit lsc-async.service has begun start-up
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- Unit lsc-async.service has begun starting up.
Jul 28 13:55:33 lsc[454816]: Use static log directory: /var/log/lsc
Jul 28 13:55:33 lsc[454862]: Error occurred during initialization of boot layer
Jul 28 13:55:33 lsc[454862]: java.lang.module.FindException: java.nio.file.AccessDeniedException: root/module-info.class
Jul 28 13:55:33 lsc[454862]: Caused by: java.nio.file.AccessDeniedException: root/module-info.class
Jul 28 13:55:33 lsc[454816]: LSC has used LOG_DIR="/var/log/lsc"
Jul 28 13:55:33 systemd[1]: lsc-async.service: Control process exited, code=exited status=1
Jul 28 13:55:33 systemd[1]: lsc-async.service: Failed with result 'exit-code'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- The unit lsc-async.service has entered the 'failed' state with result 'exit-code'.
Jul 28 13:55:33 systemd[1]: Failed to start LSC server.
-- Subject: Unit lsc-async.service has failed
-- Defined-By: systemd
-- Support: https://access.redhat.com/support
--
-- Unit lsc-async.service has failed.
--
-- The result is failed.
```

I don't have this problem when I run the command manually whether with the "root" or "lsc" user.

After trying to change different parameters in the unit, I finally found the solution. I added the "WorkingDirectory" parameter in the unit with the home directory of the lsc user (I didn't try any other value).

I propose a fix in #412

--- a/src/install/lib/systemd/system/lsc-async.service
+++ b/src/install/lib/systemd/system/lsc-async.service
@@ -22,6 +22,7 @@ ProtectKernelTunables=yes
 ProtectControlGroups=yes
 ReadWritePaths=/var/log/lsc
 UMask=0007
+WorkingDirectory=/var/lib/lsc
 
 [Install]
 WantedBy=multi-user.target

--- a/src/install/lib/systemd/system/lsc-async@.service
+++ b/src/install/lib/systemd/system/lsc-async@.service
@@ -22,6 +22,7 @@ ProtectKernelTunables=yes
 ProtectControlGroups=yes
 ReadWritePaths=/var/log/lsc
 UMask=0007
+WorkingDirectory=/var/lib/lsc
 
 [Install]
 WantedBy=multi-user.target

--- a/src/install/lib/systemd/system/lsc-sync.service
+++ b/src/install/lib/systemd/system/lsc-sync.service
@@ -21,3 +21,4 @@ ProtectKernelTunables=yes
 ProtectControlGroups=yes
 ReadWritePaths=/var/log/lsc
 UMask=0007
+WorkingDirectory=/var/lib/lsc

--- a/src/install/lib/systemd/system/lsc-sync@.service
+++ b/src/install/lib/systemd/system/lsc-sync@.service
@@ -21,3 +21,4 @@ ProtectKernelTunables=yes
 ProtectControlGroups=yes
 ReadWritePaths=/var/log/lsc
 UMask=0007
+WorkingDirectory=/var/lib/lsc

